### ISSUE TITLE
Shift+Insert for (Linux) paste primary selection possible?

### ISSUE THREAD

**Issue by dontdieych** [onivim/oni#2630]
**Oni Version: 0.36**
**Neovim Version (Linux only): 0.3.1**
**Operating System: Linux (Arch)**

**Issue:**
In Linux almost application support for paste primary selection(or clipboard) with `Shift+Insert`. Is this possible in oni?

**Comment by badosu** [comment]
Yes, it is. The clipboard we use just need to be set to retrieve the 'selection' content.

Meanwhile, you can use `<C-r>*` in insert mode or `*+p` in normal mode.
